tools/installer/requirements.txt

```diff
@@ -1,3 +1,5 @@
+git+git://github.com/exaile/pywin32-ctypes.git@pyinstaller#egg=pywin32-ctypes
+git+git://github.com/dangmai/pyinstaller.git@msys2#egg=PyInstaller
 musicbrainzngs
 mutagen==1.31
```

Comment:
We should bump the mutagen requirement.

Have you tried running this from scratch? I suspect that pyinstaller and pywin32-ctypes will need to be installed separately, as pip doesn't perform the install in order IIRC, so pyinstaller will add pywin32 to the dependency list.